NotadogApi/Domain/Models/UserUpdatePayload.cs

```diff
@@ -0,0 +1,12 @@
+using System.Collections.Generic;
+using System.Runtime.Serialization;
+
+namespace NotadogApi.Domain.Models
+{
+    public class UserUpdatePayload
```

Comment:
UserSignupCredentials have the same properties. Is it possible to redesign Models so as to remove duplication?

Reply:
hmm, I think they are different models in business-logic context.